_integration-test/run.sh

```diff
@@ -135,3 +135,10 @@ source ./custom-ca-roots/setup.sh
 $dcr --no-deps web python3 /etc/sentry/test-custom-ca-roots.py
 source ./custom-ca-roots/teardown.sh
 echo "${_endgroup}"
+
+for service in $(docker compose ps --format json | jq -r '.[]| select(.State != "running") | .Service'); do
```

Comment:
We're guaranteed to have `jq` installed?

Reply:
Yes, the unit tests require it already